docs/concepts/storage/persistent-volumes.md

```diff
@@ -711,6 +713,26 @@ parameters:
 * `location`: Azure storage account location. Default is empty.
 * `storageAccount`: Azure storage account name. If storage account is not provided, all storage accounts associated with the resource group are searched to find one that matches `skuName` and `location`. If storage account is provided, it must reside in the same resource group as the cluster, and `skuName` and `location` are ignored.
 
+`New Azure Disk Storage Class (starting from v1.7.2)`
+
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: sharedhdd
+provisioner: kubernetes.io/azure-disk
+parameters:
+  skuName: Standard_LRS
+  kind: Shared
```

Comment:
Actually it's the same meaning, user could specify skuName or storageaccounttype, do you think we need to storageaccounttype in the new config?